packages/web/src/containers/home-swap-container/HomeSwapContainer.tsx

```diff
@@ -131,10 +131,18 @@ const HomeSwapContainer: React.FC = () => {
   ]);
 
   const swapNow = useCallback(() => {
+    const direction = (() => {
+      if (tokenAAmount) return "EXACT_IN";
+
+      if (tokenBAmount) return "EXACT_OUT";
+
+      return "EXACT_IN";
+    })();
```

Comment:
What is the difference between a variable in swapDirection and a variable in direction?

Reply:
In this page we don't need to change swapDirection, it basically show tokenA, tokenB and their info, no data for direction needed here, so my my `direction` still can determine the direction when we move to swap page (we can remove swapDirection as necessary state)